src/Stratis.Features.Collateral/JoinFederationRequestMonitor.cs

```diff
@@ -41,8 +37,6 @@ public JoinFederationRequestMonitor(VotingManager votingManager, Network network
 
         public Task InitializeAsync()
         {
-            this.blockConnectedToken = this.signals.Subscribe<BlockConnected>(this.OnBlockConnected);
```

Comment:
I dont think this should be removed as we still depend on the block connected event to call `OnBlockConnected`?

Reply:
Lol Wtf. Right you are.